cmd/algoh/blockstats.go

```diff
@@ -34,20 +34,21 @@ type blockstats struct {
 func (stats *blockstats) init(block uint64) {
 }
 
-func (stats *blockstats) onBlock(block v1.Block) {
+func (stats *blockstats) onBlock(block rpcs.EncodedBlockCert) {
 	now := time.Now()
+	blockHeader := block.Block.BlockHeader
 
 	// Ensure we only create stats from consecutive blocks.
-	if stats.lastBlock+1 != block.Round {
-		stats.lastBlock = block.Round
+	if stats.lastBlock+1 != uint64(blockHeader.Round) {
+		stats.lastBlock = uint64(blockHeader.Round)
 		stats.lastBlockTime = now
 		return
 	}
 
 	// Grab unique users.
 	users := make(map[string]bool)
-	for _, tx := range block.Transactions.Transactions {
-		users[tx.From] = true
+	for _, tx := range block.Block.Payset {
+		users[tx.Txn.Sender.String()] = true
```

Comment:
@algochoi Optional - The lack of test coverage makes me a _little_ nervous.  I realize the lack of coverage predates the PR.  

Optionally, I propose extending tests to defend against an unintentional change to transaction count.  I didn't look closely, but I'm imagining `txntest.Txn` can be leveraged.

If you prefer as is, feel welcomed to resolve.

Reply:
I think the concern is warranted. I opened up a PR here to add coverage for event details here: https://github.com/algochoi/go-algorand/pull/26

Reply:
No worries, I'll resolve since the tests were merged and tests are passing.